src/test/java/chambre/morsecode/WordleControllerTest.java

```diff
@@ -0,0 +1,83 @@
+package chambre.morsecode;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import javax.swing.*;
+import java.awt.*;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+class WordleControllerTest {
+    private WordleGame game = mock();
+    private WordleDictionary dictionary = mock();
+    private JLabel[][] labels = new JLabel[][]{{
+            mock(), mock(), mock(), mock(), mock(),
+    }, {
+            mock(), mock(), mock(), mock(), mock(),
+    }, {
+            mock(), mock(), mock(), mock(), mock(),
+    }, {
+            mock(), mock(), mock(), mock(), mock(),
+    }, {
+            mock(), mock(), mock(), mock(), mock(),
+    }, {
+            mock(), mock(), mock(), mock(), mock(),
+    },
+    };
+    private JButton enter = mock();
+    private JButton back = mock();
+
+    @Test
+    public void addLetter() {
+        //given
+        WordleController controller = new WordleController(game, dictionary, labels, enter, back);
+
+        //when
+        controller.addLetter("A");
+
+        //then
+        verify(labels[0][0]).setText("A");
+        assertEquals("A", controller.theGuess.toString());
+    }
+
+    @Test
+    public void enterGuess() {
+        //given
+        WordleController controller = new WordleController(game, dictionary, labels, enter, back);
```

Comment:
You need to "set up" the mocks using doReturn so that they return the values you expect.